descriptor/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "aragon-template-descriptor-util",
```

```suggestion
  "name": "@aragon/template-descriptor",
```

And we might as well add the "private: true" declaration here too for now.